geospaas_harvesting/ingesters.py

```diff
@@ -310,35 +319,58 @@ def _get_xml_namespace(self, root):
             self.LOGGER.warning('Could not find XML namespace while reading DDX metadata')
         return namespace_prefix
 
-    def _extract_global_attributes(self, root):
-        """Extracts the global attributes of a dataset from a DDX document"""
+    def _extract_attributes(self, root):
+        """
+        Extracts the global or specific attributes of a dataset or specific ones from a DDX document
+
+        "x_path_global" is pointing to the 'NC_GLOBAL' part of response of HTML to obtain general
+        information.
+        "x_path_specific" is adaptive part (or many parts) of response of HTML to obtain the
+        raw names of parameters for cumulative normalization by metanorm package as the core of the
+        harvesting processing.
```

Comment:
Try to keep it short and simple, and as much as possible avoid including information that doesn't directly concern this method.
```suggestion
        "x_path_specific" is used to extract the dataset parameter names from the DDX document
```

Reply:
Done in the next commit